Makefile

```diff
@@ -1,6 +1,6 @@
 NAME       := mercury
 SRC_EXT    := bz2
-DL_VERSION := 2.0.0rc3
+DL_VERSION := 2.0.0
```

Comment:
Is this necessary any more?

Reply:
i kept it initially in case we have 2.0.1rc1 or some such later, but removed now